addon-test-support/extend/find-many.js

```diff
@@ -0,0 +1,36 @@
+import { getExecutionContext } from '../-private/execution_context';
+
+/**
+ * @public
+ *
+ * Returns array of elements
+ *
+ * @example
+ *
+ * import { findMany } from '../extend';
+ *
+ * export default function count(selector, options = {}) {
+ *   return {
+ *     isDescriptor: true,
+ *
+ *     get() {
+ *       return findMany(this, selector, options).length;
+ *     }
+ *   };
+ * }
+ *
+ * @param {Ceibo} pageObjectNode - Node of the tree
+ * @param {string} targetSelector - Specific CSS selector
+ * @param {Object} options - Additional options
+ * @param {boolean} options.resetScope - Do not use inherited scope
+ * @param {string} options.contains - Filter by using :contains('foo') pseudo-class
+ * @param {number} options.at - Filter by index using :eq(x) pseudo-class
+ * @param {boolean} options.last - Filter by using :last pseudo-class
+ * @param {boolean} options.visible - Filter by using :visible pseudo-class
+ * @param {string} options.testContainer - Context where to search elements in the DOM
+ * @return {Array} of Element
+ */
+export function findMany(pageObjectNode, targetSelector, options = {}) {
+  const opts = Object.assign({}, options, { multiple: true });
+  return getExecutionContext(pageObjectNode).find(targetSelector, opts).get();
```

Comment:
it looks like we still support `multiple`(which is not intended) and the rest of the props that are supported by `getExecutionContext(pageObjectNode).find()`.
I think we should take it under control by picking and passing only the sane options + test each of them.